lib/rules/global-require.js

```diff
@@ -52,9 +52,25 @@ function isShadowed(scope, node) {
 /** @type {import('../shared/types').Rule} */
 module.exports = {
     meta: {
-        deprecated: true,
-
-        replacedBy: [],
+        deprecated: {
+            message: "Node.js rules were moved out of ESLint core.",
+            url: "https://eslint.org/docs/latest/use/migrating-to-7.0.0#deprecate-node-rules",
+            deprecatedSince: "7.0.0",
+            availableUntil: null,
+            replacedBy: [
+                {
+                    message: "eslint-plugin-n now maintains deprecated Node.js-related rules",
+                    plugin: {
+                        name: "@eslint-community/eslint-plugin-n",
+                        url: "https://github.com/eslint-community/eslint-plugin-n"
+                    },
+                    rule: {
+                        name: "global-require",
+                        url: "https://github.com/eslint-community/eslint-plugin-n/tree/master/docs/rules/global-require"
```

Comment:
It looks like the `.md` extension is required for the like to GitHub files (`https://github.com/`) to work. I'm sorry if last comment was misleading:

```suggestion
                        url: "https://github.com/eslint-community/eslint-plugin-n/tree/master/docs/rules/global-require.md"
```